.github/workflows/dependabot-batcher.yml

```diff
@@ -0,0 +1,38 @@
+# Dependabot Batcher v1.0.1
+
+name: 'Dependabot Batcher'
+
+# Controls when the workflow will run
+on:
+  push:
+    branches: [ "r/12.x", "develop" ]
+  pull_request:
+    branches: [ "r/12.x", "develop" ]
```

Comment:
This would run on all pushes to the repository and opened pull requests. You probably just want to react to either of

- pushed to `dependabot/**` branches
- pull requests created or updated by dependabot
For example, something like:

```yaml
push:
  branches:
    - 'dependabot/**'
```